src/model/immutable/ContentState.js

```diff
@@ -56,6 +56,17 @@ class ContentState extends ContentStateRecord {
     return block;
   }
 
+  getFirstLevelBlocks(): BlockMap {
+    return this.getBlockChildren('');
```

Comment:
It may break compatibility if we introduce a root block with a special key.

I'm not sure how people using Draft store the contentState between editing sessions, but I think we should have as goal: "The exact data set from Draft '0.7.0' should work without changes in this version".